.circleci/config.yml

```diff
@@ -639,23 +653,41 @@ workflows:
             branches:
               only: env/dev2
 
-      - hodl:
+      - hodl_blue:
           type: approval
           filters:
             branches:
               ignore: /.*/
             tags:
               only: /^v.*$/
 
-      - deploy_uat:
+      - hodl_green:
+          type: approval
+          filters:
+            branches:
+              ignore: /.*/
+            tags:
+              only: /^v.*$/
+
+      - deploy_uat_blue:
           requires:
             - eunit
             - static_analysis
             - linux_package
-            - hodl
+            - hodl_blue
           filters:
             branches:
-              ignore: /.*/
+              only: env/uat_blue
+            tags:
+              only: /^v.*$/
+
+      - deploy_uat_green:
+          requires:
+            - deploy_uat_blue
+            - hodl_green
+          filters:
+            branches:
+              only: env/uat_green
```

Comment:
How can this (I mean triggering job `deploy_uat_green` by pushing branch `env/uat_green`) work, considering that job `deploy_uat_blue` - that this job `deploy_uat_green` depends on - is not triggered on push to branch `env/uat_green`?

on which job deploy_uat_blue depends on - is not triggered?

Reply:
I decided that we don't need manual blue/green UAT triggers, because I don't see a use case to deploy random source tree on UAT, it's should be sequential following the release process.

The source code to be release could be tagged on the previous day to trigger release workflow and deploy on blue env., then on the next (release) day, the green deployment should be unhold.

see 1e3363e